EIPS/eip-7851.md

```diff
@@ -0,0 +1,245 @@
+---
+eip: 7851
+title: EOA private key deactivation/reactivation
+description: Introduce a new precompiled contract for EOAs with delegated code to deactivate or reactivate private keys.
+author: Liyi Guo (@colinlyguo)
+discussions-to: https://ethereum-magicians.org/t/eip-7851-eoa-private-key-deactivation-reactivation/22344
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-12-27
+requires: 20, 2612, 7701, 7702
```

Comment:
```suggestion
requires: 7702
```

I don't see how ERC-20, EIP 2612, and 7701 are required here. If you would require 7701 this means this EIP can only be active once EOF is also active.

Reply:
oh I see. thanks for the clarification here.